core/controllers/suggestion.py

```diff
@@ -149,6 +149,39 @@ def post(self):
         self.render_json(self.values)
 
 
+class UpdateSuggestionHandler(base.BaseHandler):
```

Comment:
@DubeySandeep Sorry for getting bit delayed for addressing this. I move the request for update suggestion to a new handler. Reasons why I did so:

1. Keeping this in the `SuggestionHandler` results to be enabled the same requests with POST requests as well.
2. If we do implement with this in the same handler, update requests will have to be filtered by parameters rather having a dedicated route. This would result a complicated implementation for suggestion creation as well as suggestion update.

Reply:
> Keeping this in the SuggestionHandler results to be enabled the same requests with POST requests as well.

Sorry, I didn't get this can you please rephrase it?

I'm fine with the new handler, I have a few questions:
1. Are we expecting `UpdateSuggestionHandler` to only handle translation suggestions? If yes, please change the handler class name and docstring to highlight this!
2. In future once we allow updating all the suggestion are we expecting to use `SuggestionHandler`?